api/v1beta1/types.go

```diff
@@ -291,3 +291,14 @@ const (
 	// AmazonLinuxGPU is the AmazonLinux GPU AMI type.
 	AmazonLinuxGPU EKSAMILookupType = "AmazonLinuxGPU"
 )
+
+// SubnetSchemaType specifies how given network should be divided on subnets
+// in the VPC depending on the number of AZs.
+type SubnetSchemaType string
```

Comment:
API changes only need to go into the newest API version (i.e. v1beta2).  So these change here can be removed.

Reply:
There will be some changes required as a result with the "round trip fuzzy" tests. I can help with these if you run into issues.